hphp/runtime/vm/jit/smashable-instr-arm.h

```diff
@@ -77,6 +77,22 @@ constexpr size_t kSmashJccTargetOff = 12;
 
 ///////////////////////////////////////////////////////////////////////////////
 
+template<typename T>
+  static void patchInstr(TCA inst, T target) {
+  *reinterpret_cast<T*>(inst) = target;
+  auto const end = reinterpret_cast<char*>(inst + sizeof(T));
+  auto const begin = end - sizeof(T);
+  __builtin___clear_cache(begin, end);
+}
+
+using TCA32 = uint32_t;
+
+template <class T>
+static uint32_t makeTCA32(T t) {
```

Comment:
You've declared these functions in a header file, so what's the point of the `static`?

Reply:
Why would including the header in multiple files require `static` for an inline template function?  Maybe I'm being dense.

Reply:
Actually, yeah. I didn't realize that since this was a template'd function that it didn't doesn't need to be explicitly marked as static or inline to avoid a linking problem. There are two other functions there, that aren't template'd and I did need to make them static. I'm about to push a patch which makes all three explicitly inline. Let me know what you think?